ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift

```diff
@@ -415,10 +420,10 @@ extension MainViewController {
             lblTerminalId.text = terminalConfigResponse.getTerminalId()
             lblTerminalModel.text = terminalConfigResponse.getTerminalModel()
             
-            showMessage(title: "Terminal Configuration", msg: "Terminal Configuration retrieving successful", type: "INFO", isShow: true)
+            //showMessage(title: "Terminal Configuration", msg: "Terminal Configuration retrieving successful", type: "INFO", isShow: true)
             return
         } else {
-            showMessage(title: "Terminal Configuration", msg: "ERROR: Terminal Configuration retrieving failed", type: "ERROR", isShow: true)
+            //showMessage(title: "Terminal Configuration", msg: "ERROR: Terminal Configuration retrieving failed", type: "ERROR", isShow: true)
```

Comment:
Seeing as these are commented out, did you just want to remove the `else` statement too?

Reply:
Ah yes, I put comments for testing and forgot to refactor before PR. I'll fix this now.